.github/workflows/release-chart.yaml

```diff
@@ -1,18 +1,34 @@
 name: Release Helm Chart
 on:
   push:
-    branches:
-      - develop
-    paths:
-      - 'chart/**'
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+**'
 
 jobs:
   release-chart:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: cachix/install-nix-action@v17
+      - name: Pre-populate nix-shell
+        run: |
+          export NIX_PATH=nixpkgs=$(jq '.nixpkgs.url' nix/sources.json -r)
+          echo "NIX_PATH=$NIX_PATH" >> $GITHUB_ENV
+          nix-shell -p semver-tool -p yq-go --run "echo"
```

Comment:
should a `--pure` be added to ensure that only `echo` from nix is used?
I checked that `-p` is not for pure but for `packages`